modmesh/pilot/__init__.py

```diff
@@ -42,11 +42,12 @@
     RManager,
     RCameraController,
 )
-from ._gui import (  # noqa: F401
-    controller,
-    launch,
-)
-from . import airfoil  # noqa: F401
+if enable:
```

Comment:
Do not import GUI code from `modmesh.pilot` namespace directly, although the sub-package is for GUI.  This is to make testing easier to organize.